examples/esp8266/esp8266.ino

```diff
@@ -189,4 +200,4 @@ void loop() {
   }
 
   delay(100);
-}
+}
```

Comment:
Removing the empty EOF breaks many old C/C++ tools, not sure if anyone uses them with an Arduino files but it's probably safer to add it back in.

Reply:
Done!